src/Core/Grid/Query/Filter/DoctrineFilterApplicator.php

```diff
@@ -80,8 +86,67 @@ public function apply(QueryBuilder $qb, SqlFilters $filters, array $filterValues
                         $qb->setParameter($name, sprintf('%s %s', $value['to'], '23:59:59'));
                     }
 
+                    break;
+                case SqlFilters::MIN_MAX:
+                    if (!isset($value['min_field']) && !isset($value['max_field'])) {
```

Comment:
Isn't this a case where you should throw a `InvalidFilterDataException` as well? If the `MIN_MAX` is requested without the minimum value it's invalid right?

In which case this can be handled in the `computeMinMaxCase` function I guess

Reply:
Well if this if was removed then yeah eventually `computeMinMaxCase` would throw and error in such case. 

Whether it should I am not sure. As an example a `WHERE_DATE` case doesn't throw an error if both `from` and `to` are not set. 
But I guess throwing an error is safer. So I am jsut gonna remove this` if isset..` here.